lib/messenger.rb

```diff
@@ -0,0 +1,16 @@
+require 'twilio-ruby'
+
+class Messenger
+  def create( from, to, body)
+    account_sid = 'AC89d950d129482b828259244e80366b30'
+    auth_token = ''
+    # set up a client to talk to the Twilio REST API
+    @client = Twilio::REST::Client.new account_sid, auth_token
```

Comment:
you could inject the client into the method to allow easy stubbing - also means if you ever change your SMS provider it is easy to replace one for the other.